docs/literature_review.md

```diff
@@ -1,6 +1,84 @@
-Last update: 10 Feb 2020
+Last update: 12 Jan 2020
 
-Overview
+Transformers review
+
+* Pre training on domain data from scratch is the best approach
+* A more computation efficient approach, if your dataset is small, is to create an expanded data from bringing the closest examples from the domain and pre training there
+* Training from scratch a smaller BERT performs much worse than distilling a bigger BERT into a smaller one
+
+# Don't stop pre training
+https://www.aclweb.org/anthology/2020.acl-main.740.pdf
```

Comment:
this is a personal thing, so feel free to ignore - but I really like to see the names of the authors and years at a glance in a literature review. So would prefer those things with a hyperlink than just the url here.

Reply:
ok so transform that to a link and add First author et al, Year?

Reply:
yeh, thatd be my preference anyway